src/plugins/links/public/components/editor/link_destination.tsx

```diff
@@ -12,24 +12,24 @@ import { DashboardContainer } from '@kbn/dashboard-plugin/public/dashboard_conta
 
 import { EuiFormRow } from '@elastic/eui';
 import {
-  NavigationLinkType,
+  LinkType,
   EXTERNAL_LINK_TYPE,
   DASHBOARD_LINK_TYPE,
 } from '../../../common/content_management';
-import { NavigationEmbeddableUnorderedLink } from '../../editor/open_link_editor_flyout';
+import { UnorderedLinks } from '../../editor/open_link_editor_flyout';
 import { ExternalLinkDestinationPicker } from '../external_link/external_link_destination_picker';
 import { DashboardLinkDestinationPicker } from '../dashboard_link/dashboard_link_destination_picker';
-import { NavEmbeddableStrings } from '../navigation_embeddable_strings';
+import { LinksStrings } from '../links_strings';
 
-export const NavigationEmbeddableLinkDestination = ({
+export const LinkDestination = ({
   link,
   setDestination,
   parentDashboard,
   selectedLinkType,
 }: {
-  selectedLinkType: NavigationLinkType;
+  selectedLinkType: LinkType;
   parentDashboard?: DashboardContainer;
-  link?: NavigationEmbeddableUnorderedLink;
+  link?: UnorderedLinks;
```

Comment:
This is a singular link, right?

```suggestion
  link?: UnorderedLink;
```